tests/foreman/api/test_contentviewversion.py

```diff
@@ -66,16 +70,65 @@ def test_delete_version(self):
         content_view = content_view.read()
         # Get published content-view version id
         self.assertEqual(len(content_view.version), 1)
-        # Get 'Library' life-cycle environment id
-        response = client.get(
-            entities.LifecycleEnvironment().path(),
-            auth=get_server_credentials(),
-            data={u'organization_id': org.id},
-            verify=False,
-        )
-        response.raise_for_status()
-        lc_env_id = response.json()['results'][0]['id']
+        cvv = content_view.version[0].read()
+        self.assertEqual(len(cvv.environment), 1)
         # Delete the content-view version from selected env
-        content_view.delete_from_environment(lc_env_id)
+        content_view.delete_from_environment(cvv.environment[0].id)
         # Delete the version
         content_view.version[0].delete()
+        # Make sure that content view version is really removed
+        content_view = content_view.read()
+        self.assertEqual(len(content_view.version), 0)
```

Comment:
```content_view``` variable won't be used anywhere else. You can drop it with:
```python
    self.assertEqual(len(content_view.read().version), 0)
```
Same for the rest tests.